app/src/main/resources/migrering/V0.52__meldekort_tabell.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE meldekort
+(
+    id SERIAL PRIMARY KEY,
+    behandling_id BIGINT NOT NULL REFERENCES behandling (id) ON DELETE CASCADE,
+    journalpost_id VARCHAR(50) NOT NULL UNIQUE
+);
+
+CREATE TABLE arbeid_i_periode (
+    id SERIAL PRIMARY KEY,
+    periode daterange NOT NULL,
+    timerArbeidet NUMERIC(5,2) NOT NULL,
```

Comment:
```suggestion
    timerArbeidet integer NOT NULL,
```

Kan kun være heltall